substrate/frame/multisig/src/lib.rs

```diff
@@ -50,23 +50,9 @@ pub mod weights;
 
 extern crate alloc;
 use alloc::{boxed::Box, vec, vec::Vec};
-use codec::{Decode, Encode, MaxEncodedLen};
-use frame_support::{
-	dispatch::{
-		DispatchErrorWithPostInfo, DispatchResult, DispatchResultWithPostInfo, GetDispatchInfo,
-		PostDispatchInfo,
-	},
-	ensure,
-	traits::{Currency, Get, ReservableCurrency},
-	weights::Weight,
-	BoundedVec,
-};
-use frame_system::{self as system, pallet_prelude::BlockNumberFor, RawOrigin};
-use scale_info::TypeInfo;
-use sp_io::hashing::blake2_256;
-use sp_runtime::{
-	traits::{BlockNumberProvider, Dispatchable, TrailingZeroInput, Zero},
-	DispatchError, RuntimeDebug,
+use frame::{
+	prelude::*,
+	traits::{BlockNumberProvider, Currency, ReservableCurrency},
```

Comment:
`BlockNumberProvider` should go into the prelude, as it is quite useful to almost all pallets?